### DISCUSSION TITLE
Latest 2.0.9 release - docker image availability

### DISCUSSION THREAD

**Discussion by alexdelprete** [Lissy93/dashy#669] [Help and Support] [answered]
Hi Lissy,

usually when I receive the notification of a new release and I try to pull from docker the latest, everything's ok and the upgrade is quick and smooth.

After the 2.0.9 announcement, I waited a couple of hours then I tried to upgrade but there was no new image available. I thought it was only a matter of waiting so I tried today, after almost a day, but it's still not available.

Maybe it's a normal thing, I thought the release process to docker or ghcr.io was quicker.

If it's not an issue please pardon me for asking, I thought maybe there could be a problem you didn't notice in the publishing process.

Thanks for all your great work,

Alessandro

**Comment by lissy93** [comment, marked as answer]
Yeah I wrote an [automation](https://github.com/Lissy93/dashy/blob/master/.github/workflows/docker-build-publish.yml) for this a while back, so usually happens automatically. Looks like the latest run [failed](https://github.com/Lissy93/dashy/actions/workflows/docker-build-publish.yml), probably because the yarn.lock wasn't updated I think

**Comment by alexdelprete** [comment]
Everything ok now, just upgraded. Thanks for your support.
